settlement/tests/treasury.rs

```diff
@@ -12,37 +12,51 @@ use std::cell::RefCell;
 use std::collections::HashMap;
 use std::rc::Rc;
 
+fn string_to_hex(s: &str) -> HexSerializedVec {
```

Comment:
It makes the same result as before except for type. 

```decode_hex``` would be better when we handle hexadecimal strings.

Reply:
Yes, but I mean, if we want to decode the hex string to Vec(e.g.: "11111111" -> [17, 17, 17, 17]), it doesn't work. But I think it isn't a problem here.